app/src/prod/java/cm/aptoide/pt/app/view/AppSupportsBillingInfoViewHolder.java

```diff
@@ -0,0 +1,16 @@
+package cm.aptoide.pt.app.view;
+
+import android.view.View;
+import android.widget.LinearLayout;
+
+public class AppSupportsBillingInfoViewHolder {
+  private LinearLayout appcBillingSupported;
+
+  public AppSupportsBillingInfoViewHolder(LinearLayout appcBillingSupported) {
+    this.appcBillingSupported = appcBillingSupported;
+  }
+
+  public void showInfo() {
+    this.appcBillingSupported.setVisibility(View.GONE);
```

Comment:
The problem with this is that I think by changing the name to something that would seem more correct it would be something like: handleInformationVisibility... That would decrease the readability in the view compared to something so direct as showInfo(). I do agree that it is not the best name because it will hide flavor implementations that actually set it to not visible, but I cannot think of a better idea right now.